modules/transactions/src/test/java/org/apache/ignite/internal/tx/storage/state/TxStateStorageAbstractTest.java

```diff
@@ -159,11 +167,29 @@ public void testScan() throws Exception {
         }
     }
 
+    private IgniteBiTuple<UUID, TxMeta> putRandomTxMetaWithCommandIndex(TxStateStorage storage, int enlistedPartsCount, long commandIndex) {
+        UUID txId = UUID.randomUUID();
+        TxMeta txMeta = new TxMeta(TxState.PENDING, generateEnlistedPartitions(enlistedPartsCount), generateTimestamp(txId));
+        storage.compareAndSet(txId, null, txMeta, commandIndex);
+
+        return new IgniteBiTuple<>(txId, txMeta);
+    }
+
     private static void assertTxMetaEquals(TxMeta txMeta0, TxMeta txMeta1) {
         assertEquals(txMeta0.txState(), txMeta1.txState());
         assertEquals(txMeta0.commitTimestamp(), txMeta1.commitTimestamp());
         assertEquals(txMeta0.enlistedPartitions(), txMeta1.enlistedPartitions());
     }
 
-    protected abstract TxStateStorage createStorage();
+    /**
+     * Creates {@link TxStateStorage} to test.
+     *
+     * @return Tx state storage.
+     */
+    protected abstract TxStateTableStorage createStorage();
+
+    /**
+     * Destroy storage, for proper clean-up.
+     */
+    protected abstract void destroyStorage();
```

Comment:
What's the point of the abstract method? Inheritors can do it in their `afterTest` if they need it